RELEASE.md

```diff
@@ -1,3 +1,37 @@
+# Release 1.7.0
+
+The 1.7 minor series tracks TensorFlow 1.7.
+
+## Hightlights
+
+- (Beta) New Beholder plugin that shows a live video feed of tensor data during
```

Comment:
It should be OK to remove `(Beta)`. It works. We nor Chris are able to do more than maintenance on Beholder at the moment.

Reply:
That's why I prefer having the `(Beta)` label there actually.  I think it makes it clearer that it's not intended to be fully supported to the same extent as e.g. the scalars dashboard.  Also, while it works, there are certainly still robustness issues and edge cases (like the file permissions problem I just fixed) and I think it's worth making that clearer to users.

Reply:
[Optional] The wording 'New ...' might be enough to communicate it's not yet been battle-tested. We'll maintain Beholder going forward. That may involve help from the community.